quick-start.md

```diff
@@ -0,0 +1,85 @@
+# Quick Start Guide
+
+#### k8s-kurated-addons-cli
+
+This is a guide to help developers start using this repository. Even though it's based on a cluster spawned by k8s-kurated-addons, as long as the destination cluster has knative, the CLI will work.
+
+### Prerequisites
+
+- The project's dependencies as described in [the README](README.md)
+- `tilt` (if using `k8s-kurated-addons` to bring up the cluster)
+
+### Using the software
+
+Follow the steps:
+
+1. Install kka-cli (run `make build` and copy the executable to PATH)
+2. If you want to use `k8s-kurated-addons`, clone it and run `make`
+    * this will bring up a cluster using kind (kubernetes in Docker)
+    * if you want to use ngrok, you'll need to edit the `manifests/kind/templates/cluster.yaml` and add the `certSANs:` section to the `apiServer` configuration, like this:
+      ```
+      ...
+      kubeadmConfigPatches:
+      - |-
+        kind: ClusterConfiguration
+        apiServer:
+          certSANs:
+            - "127.0.0.1"
+            - "localhost"
+            - "*.tcp.eu.ngrok.io"
+            - "*.tcp.sa.ngrok.io"
+            - "*.tcp.na.ngrok.io"
+      ...
+      ```
+3. Wait for the cluster to stabilize
+    * it's easier to spot checking `tilt` and `argocd` (they will show everything green)
+4. Create a repo and enable read and write permissions for the GitHub Actions workflows as described in the [GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions)
+5. Clone the new repo
+6. Get argocd-password, enable and access the argocd port forward from Tilt
+7. Create a new branch from main / master in the repo
+8. Run `kka-cli --app-name <app name> init config > .kka`
+    * the app name needs to be unique since it will be used by knative to expose its service
+    * it’s going to be on the domain GitHub Actions will output
+    * if the organization name or GitHub account has uppercase characters, you will need to edit the `.kka` file and change the repo-name to a fully lowercase string
+9. Run `kka-cli init github`
+10. Replace `latest` with `kka-feature-improve-workflow` in `.github/workflows/kka_onmain.yml` and `.github/workflows/kka_onbranch.yml`
```

Comment:
This part should be fine now since the branch was merged yesterday, we can remove the step